pkg/services/ngalert/notifier/channels/email.go

```diff
@@ -95,6 +98,18 @@ func (en *EmailNotifier) Notify(ctx context.Context, as ...*types.Alert) (bool,
 	return true, nil
 }
 
+func getTitleFromTemplateData(data *template.Data) string {
+	title := "[" + data.Status
+	if data.Status == string(model.AlertFiring) {
+		title += ":" + fmt.Sprintf("%d", len(data.Alerts.Firing()))
+	}
+	title += "]" + strings.Join(data.GroupLabels.SortedPairs().Values(), " ") + " "
+	if len(data.CommonLabels) > len(data.GroupLabels) {
+		title += "(" + strings.Join(data.CommonLabels.Remove(data.GroupLabels.Names()).Values(), " ") + ")"
+	}
+	return title
+}
+
```

Comment:
Since we are using Prometheus style alerts, this title is inspired from Prometheus Alertmanager.

https://github.com/prometheus/alertmanager/blob/10757eb5fb78c5c58586812721e9833b0d8500ee/template/default.tmpl#L4